conf/documentation.conf

```diff
@@ -1875,3 +1875,25 @@ options=enabled|disabled
 description=<<EOT
 Show the parking portal to the device instead of the usual portal
 EOT
+
+[radius_authentication_methods.eap_authentication_types]
+type=multi
+options=GTC|MD5|MSCHAPv2|PEAP|FAST|LEAP
```

Comment:
So remove LEAP completely or keep it there since it is configured in 
freeradius?

James Rouzier
jrouzier@inverse.ca :: +1.514.447.4918 (x115)  ::  http://www.inverse.ca
Inverse inc. :: Leaders behind SOGo (http://www.sogo.nu) and PacketFence (http://www.packetfence.org)

On 2016-08-12 4:10 PM, extrafu wrote:
>
> In conf/documentation.conf 
> <https://github.com/inverse-inc/packetfence/pull/1655#discussion_r74653263>:
>
> > @@ -1875,3 +1875,25 @@ options=enabled|disabled
> >  description=<<EOT
> >  Show the parking portal to the device instead of the usual portal
> >  EOT
> > +
> > +[radius_authentication_methods.eap_authentication_types]
> > +type=multi
> > +options=GTC|MD5|MSCHAPv2|PEAP|FAST|LEAP
>
> I think we should eventually. IIRC there are stuff to configure for 
> doing EAP-SIM that are outside of the scope of PacketFence. Let's add 
> EAP-SIM in there when we actually handle this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/inverse-inc/packetfence/pull/1655/files/94e590377ea95e45c4419f5bed0d55bea7895e5d#r74653263>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ACmeMUSucqIZ7vJ-koH-51HbbX5ieFT2ks5qfNM5gaJpZM4Jjaaz>.
>